docs/content/latest/deploy/public-clouds/aws/manual-deployment.md

```diff
@@ -578,6 +578,60 @@ replication_info {
 }
 ```
 
+By default, all nodes are eligible to have tablet leaders. 
+To get better transactional performance and lower latency, it's better that all tablet-leaders reside in the same region if possible.
+This will reduce the number of cross-region hops when writing transactions.
```

Comment:
Move a simple example to yb-admin.md page. Don't add an example here.

Here instead you can say something like:

Note: This example is a multi-AZ (single region deployment). But suppose you are using these instructions to setup a multi-region deployment, one option to consider is to set a preferred location for all the tablet leaders using the `set_preferred_zones` yb-admin command (add link here to the yb-admin.md relevant section). For multi-row/multi-table transactional operations, colocating the leaders to be in a single zone/region can help reduce the number of cross-region network hops involved in executing the transaction and as a result improve performance.

And remove the rest of the changes.

Reply:
fixed in https://github.com/yugabyte/yugabyte-db/pull/2844/commits/f079ca761cc6668dad974a9155b3a4ea7203cd69